src/boot/store.js

```diff
@@ -9,13 +9,24 @@ import middleware from './middleware';
 
 // AsyncStorage.clear(); // use to reset storage during development
 
-// uncomment the following lines to integrate reactotron with redux
-// const store = Reactotron.createStore(
-//   rootReducer,
-//   compose(autoRehydrate(), applyMiddleware(...middleware)),
-// );
+let store;
 
-const store = compose(applyMiddleware(...middleware), autoRehydrate())(createStore)(rootReducer);
+if (__DEV__) { // This block of code is never meant to execute in production
```

Comment:
Moved Reactotron store creation in `__DEV__` block so that normal store is created in production environment.